.travis.yml

```diff
@@ -37,6 +37,7 @@ matrix:
       # Latest commit to master
     - php: 7.2
       env: STABILITY="dev"
+    - php: 7.3
```

Comment:
please move this up to go on the line where we have 7.2 with the heading "Test the latest stable release"